shared/macros/10-ocil.jinja

```diff
@@ -639,13 +639,14 @@ ocil_clause: '"{{{ part }}} is not a mountpoint" is returned'
 
 {{# Firewalld macros #}}
 
-{{%- macro _firewalld_check(access_action, port, proto, service) %}}
-    To determine if <code>firewalld</code> is configured to {{{ access_action }}} to <code>{{{ service }}}</code>
+{{%- macro _firewalld_check(access_action, port=none, proto=none, service=none) %}}
+    To determine if <code>firewalld</code> is configured to {{{ access_action }}}
+    {{% if port is not none %}}
```

Comment:
This is a nice fix, but let's be a bit stricter - based on the usage, it looks like that all arguments are passed, so let's make them required. The uncertainty of what behaviors should be suppoted, lack of tests and the messy nature of Jinja make those macros like small puzzles, while they should be mainly tools.

Reply:
Ok